qiita_core/util.py

```diff
@@ -96,6 +96,7 @@ def get_qiita_version():
     try:
         repo = Repo(git_repo_path)
         sha = repo.active_branch.commit.hexsha
+        repo.__del__()
```

Comment:
@antgonza this method is usually called automatically once the object falls out of scope. Is there a reason you want to explicitly call it? That seems a little odd. Maybe if you can add a comment saying why this is needed?

Reply:
This: https://github.com/qiita-spots/qiita/issues/3216#issuecomment-1297345627